src/reanimated2/core.ts

```diff
@@ -52,29 +47,7 @@ global._log = function (s: string) {
   console.log(s);
 };
 
-export function getViewProp<T>(viewTag: string, propName: string): Promise<T> {
```

Comment:
This function can still be used internally for testing purposes. We should either remove it completely (along with `NativeReanimatedModule.getViewProp` and native code) or leave it as it is now.

Reply:
removele